pkg/cmd/server/origin/master.go

```diff
@@ -194,7 +194,7 @@ func (c *MasterConfig) newOAuthServerHandler(genericConfig *apiserver.Config) (h
 }
 
 func (c *MasterConfig) withAggregator(delegateAPIServer apiserver.DelegationTarget, kubeAPIServerConfig apiserver.RecommendedConfig, apiExtensionsInformers apiextensionsinformers.SharedInformerFactory) (*aggregatorapiserver.APIAggregator, error) {
-	aggregatorConfig, err := c.createAggregatorConfig(kubeAPIServerConfig)
+	aggregatorConfig, err := c.createAggregatorConfig(kubeAPIServerConfig.Config, c.ClientGoKubeInformers)
```

Comment:
Why isn't `kubeAPIServerConfig.SharedInformerFactory` set with a non-nil value?

Reply:
This change looks suspicious, now hiding another error.

Reply:
The pattern is good. But we have the same problem with `kubeAPIServerConfig`. It should have the informer field set.